test/e2e-tests.sh

```diff
@@ -134,7 +121,7 @@ do
   if [ "$resp" != "" ]; then
 		echo "Response from pod is: $resp"
```

Comment:
I know this isn't your code but while you're here could you fix the indenting on this line please 👼